blockchain/blocks/src/ticket.rs

```diff
@@ -4,7 +4,7 @@
 use forest_crypto::VRFProof;
 use forest_encoding::tuple::*;
 
-/// A Ticket is a marker of a tick of the blockchain's clock.  It is the source
+/// A Ticket is a marker of a tick of the block-chain's clock.  It is the source
```

Comment:
Just `blockchain` is not fine? Or should we add it until we have a better dictionary?

Reply:
Try to suffix blockchain in our dictionary with `/M`, `blockchain/M`.

Reply:
> It tells hunspell to also accept the possessive version of the word.

Reply:
:+1: